docs/odissei-tutorial.md

```diff
@@ -0,0 +1,137 @@
+
+# How to use the `sirup` package to manage your IP address in python
+
+When scraping data from the internet, it can be handy to use an IP address that is different from your real IP address. The `sirup` package in python allows you to do this in a flexible way through a VPN service.
+
+**Note**
+When using IP rotationg for webscraping, it is important to respect the law and not overload target website. This means, for instance, moderating the number of requests made to a website in a given time. 
+
+**Note**
+The package works only on Linux at the moment.
+
+
+## What you need 
+
+### 1. `openvpn`
+
+[OpenVPN](https://en.wikipedia.org/wiki/OpenVPN) is a system that allows you to create secure VPN connections. You can install it by following [these instructions](https://community.openvpn.net/openvpn/wiki/OpenvpnSoftwareRepos).
+
+
+### 2. Root access to your computer
+
+Because internet connections are an important security concern, `OpenVPN` requires root access -- this is the equivalent to administrator rights on a windows computer. If you have root access, you can for instance run the following command on your terminal:
+
+```bash
+sudo ls -lh # will ask you for your root password
+``` 
+
+
+## Installing and setting up `sirup`
+
+You can install `sirup` as follows:
+
+```python
+python -m pip install sirup
+```
+
+To use the package and change your IP address, you need to connect to a VPN server, and for this, you need an account with a VPN service provider that offers OpenVPN configuration files for your account. At the time of writing, examples of such services are ProtonVPN and Surfshark. We will use ProtonVPN in this tutorial.
```

Comment:
"need to connect to" -> you mean "first need to connect to" i.e., you should start the server prior to executing any functions from the library?